RHDHBUGS-1228: Argo CD update (#2245)
redhat-developer/red-hat-developers-documentation-rhdh: modules/shared/proc-use-the-argo-cd-plugin.adoc

Reviewer: ```suggestion
```

Reviewer: ```suggestion
```

Reviewer: ```suggestion
```

This plugin provides a visual overview of the application’s status, deployment details, commit message, author of the commit, container image promoted to environment and deployment history.

.Prerequisites

> * You have xref:enable-the-argo-cd-plugin_{context}[enabled Argo CD plugin] in {product-very-short}.

Reviewer: This xref targets the anchor defined in `proc-enable-the-argo-cd-plugin.adoc`, but that module is only included in the `extend_configuring-dynamic-plugins` title (via `assembly-install-and-configure-argo-cd.adoc`). It is not included in the `extend_using-dynamic-plugins-in-rhdh` title, which is the only title that includes `proc-use-the-argo-cd-plugin.adoc`. Therefore the xref anchor cannot resolve, and XML validation fails.

```suggestion
* You have link:{configuring-dynamic-plugins-book-link}#enable-the-argo-cd-plugin_install-and-configure-argo-cd[enabled the Argo CD plugin] in {product-very-short}.
```

This plugin provides a visual overview of the application’s status, deployment details, commit message, author of the commit, container image promoted to environment and deployment history.

.Prerequisites

> * You have xref:enable-the-argo-cd-plugin_{context}[enabled the Argo CD plugin] in {product} {product-very-short}.

Reviewer: Use the expected value of the context attribute here

Author: sorry, I'm a silly goose today. what's the expected context?
